### DISCUSSION TITLE
"info: orientation"  in log file - what does it mean?

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#3176] [General]
Usually, with new iterations, I restart and check the log for anything outside of
```
info: Daemonized.
[...]
info: The Web UI can be reached by following this link:
```
, because that's a good first indicator of correctness.

Yesterday I had a bunch of subsequent lines of
` info: orientation 1`
Tried Google, found nothing.
Just curiosity: What does this tell me, please?

**Comment by KarlStraussberger** [comment]
It's a debugging message I forgot to reduce in level. It's fixed in current head.

**Comment by udippel** [comment]
Thanks for the quick answer!
